spring-xd-deployment.tex

```diff
@@ -1,30 +1,18 @@
 \section{Deployment}
 This section describes deployment architectures that are possible with Spring XD.
-Note that this is not an exhaustive list of all deployment types. See
-section~\ref{sec:Use Cases} for descriptions of real world deployments.
+While XD does not subscribe to a specific deployment model (Lambda, 
+Reactive or other), it does allow for a user to architect the deployment type 
+that meets their needs.  Note that this is not an exhaustive list of all 
+deployment types, see section~\ref{sec:Use Cases} for descriptions of real 
+world deployments.
 
 \subsection{Lambda Architecture}
 
-The Lambda Architecture, introduced by Nathan Marz \cite{lambda-architecture-paper}
-and shown in figure~\ref{fig:lambda}
-is a generic, scalable and fault tolerant data  processing architecture. It 
+One of the architectures a user can implement using XD is the
+Lambda Architecture, introduced by Nathan Marz \cite{lambda-architecture-paper}. 
+Is a generic, scalable and fault tolerant data  processing architecture that 
 attempts to provide a comprehensive solution to the problem of processing an 
-extremely large set of data.
-
-The Lambda Architecture has the following components:
```

Comment:
I would remove the Lambda architecture as a top level section. We could add a paragraph about the existence of stream and batch support in Spring XD makes lambda possible, but no more than that.